docs/03-plugins/remove-deprecated-attrs.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Remove Deprecated Attributes
+svgo:
+  pluginId: removeDeprecatedAttrs
+  defaultPlugin: true
+---
+
+Remove deprecated attributes from elements in the document.
+
+This plugin does not remove the deprecated `xlink:*` attributes. To remove them, use the `removeXlink` plugin.
```

Comment:
```suggestion
This plugin does not remove attributes from the deprecated XLink namespace. To remove them, use the [Remove XLink](/docs/plugins/remove-xlink/) plugin.
```

Can we refer to the namespace name rather than a particular prefix? (Since it can have any prefix!)

Also it'd be nicer to link to the plugins docs.